_includes/header.html

```diff
@@ -10,6 +10,15 @@
   <div class="panel-main">
     <div class="panel-main__inner panel-inverted">
       <div class="panel-main__content">
+
```

Comment:
I think I'd put it more in the default.html layout:

like around here: https://github.com/bids-apps/bids-apps.github.io/blob/3c169c74ec9cc3237200945c699f2a2fdb49e012/_layouts/default.html#L7

This way it would appear in the top of the screen rather than in the side bar.

Reply:
> I think I'd put it more in the default.html layout:
> 
> like around here:
> 
> https://github.com/bids-apps/bids-apps.github.io/blob/3c169c74ec9cc3237200945c699f2a2fdb49e012/_layouts/default.html#L7
> 
> This way it would appear in the top of the screen rather than in the side bar.

What Remi says, but if it breaks things and doesn't look right revert to [a58ad5d](https://github.com/bids-apps/bids-apps.github.io/pull/140/commits/a58ad5d404401b69bfad49c7c0d8ef9f78a0f43b) and merge it.